recipe/meta.yaml

```
@@ -35,7 +35,7 @@ requirements:
     - python
     # temporary pin pip to avoid missing oldest-supported-numpy error
     - pip ==22.0.4
-    - setuptools <60.0
+    - setuptools
     - cython >=0.29.24
```

I think Cython also must be bumped:

```suggestion
    - cython >=0.29.33
```

---

For `1.2.X`, the Cython minimum version can stay the same, which is consistent with [pyproject.toml on 1.2.X](https://github.com/scikit-learn/scikit-learn/blob/9aaed498795f68e5956ea762fef9c440ca9eb239/pyproject.toml#L6).

In general, I do not think we bump minimum dependencies on bug fix releases.

---

No, this is a bug fix release, we don't bump min dependencies for bug fix releases.